rust/Cargo.toml

```diff
@@ -10,7 +10,7 @@ members = [
 
 [workspace.package]
 authors = ["Sift Software Engineers <engineering@siftstack.com>"]
-version = "0.2.1"
+version = "0.3.0"
 edition = "2024"
```

Comment:
Should this be 2025 now?

Reply:
No this is using a very particular Rust edition, the 2024 edition :)